backend/src/main/resources/application.properties

```diff
@@ -1 +1,3 @@
 spring.application.name=backend
```

Comment:
I'm assuming this comes from an env file?
Please provide a readme to get this set up